src/netaddress.cpp

```diff
@@ -437,6 +437,11 @@ bool CNetAddr::IsValid() const
         return false;
     }
 
+    // cjdns addresses always start with 0xfc
```

Comment:
suggestion, if you retouch:
```suggestion
    // CJDNS addresses always start with 0xfc
```

as everywhere else in the file and in `net_tests.cpp` refers to CJDNS